### PR TITLE
chore: prepare v5.0.0 for release

### PR DESCRIPTION
**Pull request by Justintime50**
Prepares `v5.0.0` for release by updating the CHANGELOG, updating the version in the library (removed VERSION file as it's not being used).

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +0,0 @@
-4.0.2
```
Comment (jchen293) [inline VERSION:1]: Just curious, why is this file deleted?

**Review by jchen293 — APPROVED** [review]

**Review by nwithan8 — APPROVED** [review]
